e2e/multi-svc-app/back-end/main.go

```diff
@@ -23,17 +23,17 @@ func SimpleGet(w http.ResponseWriter, req *http.Request, ps httprouter.Params) {
 	w.Write([]byte("back-end"))
 }
 
-// ServiceDiscoveryGet just returns true no matter what
-func ServiceDiscoveryGet(w http.ResponseWriter, req *http.Request, ps httprouter.Params) {
-	log.Println("Get on ServiceDiscovery endpoint Succeeded")
+// ServiceConnectGet just returns true no matter what
+func ServiceConnectGet(w http.ResponseWriter, req *http.Request, ps httprouter.Params) {
+	log.Println("Get on service connect endpoint Succeeded")
 	w.WriteHeader(http.StatusOK)
-	w.Write([]byte("back-end-service-discovery"))
```

Comment:
Instead of changing sd to sc, what do you think of keeping both in the e2e?
1. Deploy all services using SD, test that it works
2. Update all services to use SC, test that it works